src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -517,6 +540,16 @@ var DynamicTable = new Class({
         });
     },
 
+    setRowClass: function() {
+        var that = this;
```

Comment:
Why it's needed?

Reply:
Inside the `each` function the context of `this` is changed. We need to preserve access to the original contetxt.